### PR TITLE
add Ghasedak Sms Provider

### PR DESCRIPTION
**Pull request by vessaldaneshvar**
Hi 
I add ghasedak sms provider 
main address is : https://ghasedak.me/

### CODE REVIEW

File: provider/ghasedak/ghasedak.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	return &Ghasedak{
+		config,
+		&http.Client{Timeout: time.Second * 20},
+	}
+}
+
+// Building the API and call the Ghasedak endpoint to send SMS to the configured receptor from config.yaml
+func (ns *Ghasedak) Send(message sachet.Message) error {
+	url := "https://api.ghasedak.me/v2/sms/send/pair"
+	payload := strings.NewReader("message=" + message.Text + "&receptor=" + strings.Join(message.To, ","))
```
Comment (marcelcorso) [inline provider/ghasedak/ghasedak.go:35]: There may be encoding problems here. 
You should use https://pkg.go.dev/net/url#Values.Encode

Comment (vessaldaneshvar) [reply provider/ghasedak/ghasedak.go]: I Use Encode in payload request

File: provider/ghasedak/ghasedak.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"strings"
+	"time"
+
+	"github.com/messagebird/sachet"
+)
+
+// Retrieving required data from 'ghasedak' sections of config.yaml
+type Config struct {
+	APIToken     string   `yaml:"api_token"`
+	PhoneNumbers []string `yaml:"phone_numbers"`
```
Comment (marcelcorso) [inline provider/ghasedak/ghasedak.go:17]: PhoneNumbers is not used!

Comment (marcelcorso) [reply provider/ghasedak/ghasedak.go:17]: they normally come from the `message` obj

**Comment by marcelcorso** [comment]
I added another comment for something small. Else looks good. Did you test it?

File: provider/ghasedak/ghasedak.go (hunk 2 above)
Comment (vessaldaneshvar) [reply provider/ghasedak/ghasedak.go:17]: 'PhoneNumbers' is sender phone number
I will delete this config because not usable in Ghasedak API

Comment (marcelcorso) [reply provider/ghasedak/ghasedak.go]: We call this "originator" and it's also on the Message https://github.com/messagebird/sachet/blob/master/sachet.go#L9

Comment (vessaldaneshvar) [reply provider/ghasedak/ghasedak.go]: in last pull request, I delete config

Comment (vessaldaneshvar) [reply provider/ghasedak/ghasedak.go]: in Kavenegar Provider :  Config PhoneNumbers Exist
https://github.com/messagebird/sachet/blob/master/provider/kavenegar/kavenegar.go

**Pull request merged**